packages/server/lambdas/database/get-favorite-pages.ts

```diff
@@ -0,0 +1,49 @@
+import { ALBEvent, ALBResult } from 'aws-lambda';
+
+import { getRataExtraLambdaError } from '../../utils/errors';
+import { log } from '../../utils/logger';
+import { getUser, validateReadUser } from '../../utils/userService';
+import { DatabaseClient } from './client';
+
+const database = await DatabaseClient.build();
+
+/**
+ * List user's all favorite pages. Example request: GET /api/database/favorites
+ * @param {ALBEvent} event
+ * @param {{string}} event.path Path should end with the page to get the custom content for
+ * @returns  {Promise<ALBResult>} JSON stringified object of contents inside body
+ */
+export async function handleRequest(event: ALBEvent): Promise<ALBResult> {
+  try {
+    const user = await getUser(event);
+    log.info(user, 'List favorite category pages');
+    validateReadUser(user);
+
+    const data = await database.favoriteCategory.findMany({
+      where: {
+        userId: user.uid,
+      },
+      select: {
+        id: true,
+        userId: true, // TODO: Remove after testing
```

Comment:
TODO?

Reply:
Is it bad if api returns user's id? If not, I could remove comment

Reply:
The user should know their id, so it's not a big issue. But if it's not needed, better to leave it out.